test/Microsoft.AspNetCore.Localization.FunctionalTests/project.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.1.0-*",
```

Comment:
For consistency, coz other test projects they have, either we make it for all or remove it from all